include/script.h

```diff
@@ -1,5 +1,5 @@
-#ifndef POKEPLATINUM_UNK_0203E724_H
-#define POKEPLATINUM_UNK_0203E724_H
+#ifndef POKEPLATINUM_SCRIPT_H
```

Comment:
The name here should specify that this is explicitly for field-scripting, to delineate them from other forms of scripting (AI, Battle, Frontier, etc.).

Reply:
Any suggestions for names?

Reply:
`field_script.h/c`?

Reply:
That or `field_script_context.h/c`